docs/guide/getting-started.md

```diff
@@ -26,11 +26,51 @@ if (typeof window.ethereum !== 'undefined') {
 
 You can review the full API for the `window.ethereum` object [here](./ethereum-provider.html).
 
+### Connecting to a test network.
+
+**Show test networks**<br />
+![image](https://user-images.githubusercontent.com/1038445/150907727-77fe4899-5238-4df6-8fa4-e6ae522e7d02.png)<br />
+
+![image](https://user-images.githubusercontent.com/1038445/150907769-c7ea63a8-6bd2-4f80-a376-a70243bb14b7.png)<br />
+
+**Connecting to Ropsten Test Network**<br />
+Use the network dropdown menu (which would normally be displaying Ethereum Mainnet like the image above) and select Ropsten Test Network<br />
+![image](https://user-images.githubusercontent.com/1038445/150908059-f86cd2eb-6d59-43e8-8fec-bacce425ebe0.png)<br />
+
+I recommend creating a test account just for the Ropsten Test Network. (Just give it a name)<br />
+
+![image](https://user-images.githubusercontent.com/1038445/150908294-22099c19-4b22-4587-939b-3f9f882b09e8.png)
+
+**Request 1 ETH Over the test network**<br />
+Navigate to https://faucet.metamask.io/
+Ensure you have the test account and test "Ropsten Test Network Selected"
+Click "reqest 1 ether from faucet"
+![image](https://user-images.githubusercontent.com/1038445/150909358-80beed6d-bebc-4602-90fb-183b15279228.png)
+
+A request will be sent to Metamask Client for you to sign. Click Next **WAIT Patiently for the transaction to be approved once Metamask has connected (may take a couple minutes)**
+![image](https://user-images.githubusercontent.com/1038445/150909613-6cb6d54e-3d43-4505-b8d6-7fa6b225e355.png)
+
+![image](https://user-images.githubusercontent.com/1038445/150909688-211f3398-e737-42db-bee2-e62d5e2ef5a8.png)
+
+![image](https://user-images.githubusercontent.com/1038445/150910232-703e1c26-4bdb-41aa-a174-7c10a9f3a6f8.png)
+
+![image](https://user-images.githubusercontent.com/1038445/150910318-b6cd9eb6-56e8-46b0-9664-82a072d3b0a5.png)
+
 ### Running a Test Network
 
-In the top right menu of MetaMask, select the network that you are currently connected to. Among several popular defaults, you'll find `Custom RPC` and `Localhost 8545`. These are both useful for connecting to a test blockchain, like [ganache](https://www.trufflesuite.com/ganache). You can quickly install and start Ganache if you have `npm` installed with `npm i -g ganache-cli && ganache-cli`.
+If you would like to run your own Network on you local machine. [Ganache](https://www.trufflesuite.com/ganache) has some great features for starting your application with different states.
+
+You can quickly install and start Ganache if you have `npm` installed with `npm i -g ganache-cli && ganache-cli`.
+Once Ganache is started connect to network `Localhost 8545` at the bottom of the network list (if you don't see `Localhost 8545` in your network list follow the instructions above [`Connecting to a test network`](#Connecting) to `show` test networks).
+You can now 
+![image](https://user-images.githubusercontent.com/1038445/150916135-c9d019ad-86f3-474d-a0ef-d53dabe799f9.png)
```

Comment:
I don't feel like this screenshot adds much value, especially when connected to the next paragraph about using `-m`. What do you think about removing it?